content/en/docs/release-oversight/payload-testing.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Payload Testing"
+description: An overview of payload testing.
+---
+
+TRT needs the capability to run a selected subset of release qualification jobs on selected pull requests in all repositories that contribute to OCP, before they are merged.
```

Comment:
Could be nice to link to the doc about the release qualification jobs

Reply:
updated